src/expr.rs

```diff
@@ -3,32 +3,32 @@ use std::cmp::min;
 
 use itertools::Itertools;
 use rustc_ast::token::{Delimiter, Lit, LitKind};
-use rustc_ast::{ForLoopKind, MatchKind, ast, ptr, token};
+use rustc_ast::{ast, ptr, token, ForLoopKind, MatchKind};
```

Comment:
I don't think was rebased correctly. The imports should not have changed.

Reply:
I know, I am resolving some merge conflicts. Give me a few minutes

Reply:
So the mismatch was caused because I am using autoformat with a rustfmt I installed via rustup. 

Is there a way for me install nightly rustfmt?